src/System.Private.Xml/src/System/Xml/BinaryXml/SqlUtils.cs

```diff
@@ -31,13 +31,13 @@ public bool IsPositive
             }
         }
 
-        private static readonly byte s_NUMERIC_MAX_PRECISION = 38;            // Maximum precision of numeric
-        private static readonly byte s_maxPrecision = s_NUMERIC_MAX_PRECISION;  // max SS precision
+        private const byte s_NUMERIC_MAX_PRECISION = 38;            // Maximum precision of numeric
+        private const byte s_maxPrecision = s_NUMERIC_MAX_PRECISION;  // max SS precision
```

Comment:
In other files you renamed the consts to PascalCase.  Why not these?

Reply:
I started to, and then I undid it in files where it led to complication; a bunch of these end up using hungarian notation, and in some cases the only difference between names is the prefix.  So when I could easily rename all of them in the file, I did, but otherwise I left them named as is.